client/src/App.css

```diff
@@ -52,24 +52,42 @@
   grid-row-end: 3;
 }
 
+#new-project-form,
 #overlay {
-  position: fixed;
   top: 0;
   bottom: 0;
   right: 0;
   left: 0;
+}
+
+#overlay {
+  position: fixed;
   background-color: var(--grey-90-a20);
 }
 
 #new-project-form {
   position: absolute;
   margin: auto;
-  width: 50%;
-  height: 50%;
-  left: 25%;
-  top: 25%;
+  width: 600px;
+  height: 300px;
   background-color: var(--grey-10);
-  border: 1px var(--grey-80) solid;
-  border-radius: 2px;
+  border: 1px var(--grey-60) solid;
+  border-radius: 3px;
   padding: 20px;
```

Comment:
Should we add some CSS linting/order to the properties?

Reply:
yes, not in this PR I've been wanting to bring one in #1